packages/lesswrong/lib/collections/posts/schema.tsx

```diff
@@ -2569,42 +2569,6 @@ const schema: SchemaType<"Posts"> = {
     },
   },
 
-  // GraphQL only field that resolves based on whether the current user has closed
-  // this posts author's moderation guidelines in the past
-  showModerationGuidelines: {
```

Comment:
This is a resolver only field that assumes `moderationGuidelines` is denormalized onto the posts object. This is fixable, but it turns out that this isn't actually used anywhere in the codebase so I just removed it instead.